src/components/SidebarWithHeader.tsx

```diff
@@ -165,9 +184,17 @@ const NavItem = ({ icon, children, route, onClose, ...rest }: NavItemProps) => {
                 borderRadius='lg'
                 role='group'
                 cursor='pointer'
+                color={isSelected() ? 'white' : 'black'}
+                bg={isSelected() ? 'primary' : 'white'}
+                opacity={isSelected() ? 1 : 0.5}
                 _hover={{
-                    bg: 'cyan.400',
+                    bg: 'primary',
                     color: 'white',
+                    opacity: isSelected() ? 1 : 0.5,

```

Comment:
i don't think this opacity should be set. check with @nsthi  though but i'm not a fan. should just be "1" and never "0.5"

Reply:
Updated the buttons with a colorScheme which should control the behavior of the buttons a little better.